nexus/bin/qmca

```diff
@@ -2677,6 +2683,18 @@ QMCA examples:
             #end for
         #end for
         opt = self.options
+        if opt.average and opt.save_average:
+            for fmap in self.file_map:
+                ndata = len(fmap.data['LocalEnergy'])
```

Comment:
Use fmap.data.first() instead.  Depending on the method, LocalEnergy may or may not be the key used for the total energy.